packages/plugins/test/mocha/mock.ts

```diff
@@ -10,17 +11,24 @@ import {
 import sinon from 'sinon'
 import { Operation } from '@sphinx-labs/contracts'
 
-import { readContractArtifact } from '../../src/foundry/utils'
 import { propose } from '../../src/cli/propose'
 import { deploy } from '../../src/cli/deploy'
 import { makeSphinxContext } from '../../src/cli/context'
+import { readContractArtifact } from '../../dist'
+
+const file = readdirSync(`./out/artifacts/build-info/`).at(0)
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const buildInfo = JSON.parse(
+  readFileSync(`./out/artifacts/build-info/${file}`, 'utf8')
+)
```

Comment:
See comment above, we read this in directly now since it's faster than using the `makeGetConfigArtifacts` function.